docs/env.md

```diff
@@ -172,7 +172,7 @@ A key ID + secret pair for an AWS IAM account with administrative access to the
 - Required: Yes
 - Default: None
 
-#### DJANGO_AWS_DEFAULT_REGION
+#### DJANGO_AWS_S3_REGION_NAME
```

Comment:
IMO this modification is not needed and it is a BC BREAK. Is it possible to create an alias maybe ? And then remove this variable in a next major version ?

Reply:
I think you're right. I will add a deprecation warning when this variable is set and we can remove it in the next big release.